bin/generate-manifest.js

```diff
@@ -14,7 +14,7 @@ const manifest = Object.assign(packageMeta.webextensionManifest, {
       'id': packageMeta.id
```

Comment:
Should we also change this to have the `packageMeta.webextensionManifest.applications.gecko` come after the `'id'`, to be consistent with line 17?

Reply:
Hmm, also here I think I thought having the package.json ID override the webextensionManifest data was a good idea. But I can't remember why - seems like a bad idea right now